### PR TITLE
Evaluation bar flip

### PR DESCRIPTION
**Pull request by debariev**
Evaluation bar now flips with a board, as at lichess or chess.com.

### CODE REVIEW

File: src/public/pages/report/scripts/evalbar.ts [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
         let height = Math.max(Math.min(360 - evaluation.value / 3, 680), 40);
-        evaluationBarCtx.fillRect(0, 0, 30, height);
-        
         let evaluationText = Math.abs(evaluation.value / 100).toFixed(1);
         let evaluationTextWidth = evaluationBarCtx.measureText(evaluationText).width;
+        let evaluationTextY = 0
+
+        if (boardFlipped) {
+            evaluationTextY = evaluation.value >= 0 ? 20 : 710;
+            evaluationBarCtx.fillRect(0, 720-height, 30, height);
```
Comment (Levsha1337) [inline src/public/pages/report/scripts/evalbar.ts:16]: You should place spaces before and after minus

File: src/public/pages/report/scripts/evalbar.ts [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
-        evaluationBarCtx.fillStyle = evaluation.value >= 0 ? "#1e1e1e" : "#ffffff"; 
         evaluationBarCtx.fillText(evaluationText, 15 - evaluationTextWidth / 2, evaluationTextY, 30);
     } else {
         let evaluationText = "M" + Math.abs(evaluation.value).toString();
         let evaluationTextWidth = evaluationBarCtx.measureText(evaluationText).width;
 
         if (evaluation.value > 0) {
             evaluationBarCtx.fillStyle = "#1e1e1e";
-            evaluationBarCtx.fillText(evaluationText, 15 - evaluationTextWidth / 2, 710, 30);
+            evaluationBarCtx.fillText(evaluationText, 15 - evaluationTextWidth / 2, boardFlipped? 20 : 710, 30);
```
Comment (Levsha1337) [inline src/public/pages/report/scripts/evalbar.ts:31]: Same here: space before question mark

File: src/public/pages/report/scripts/evalbar.ts [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
         } else if (evaluation.value < 0) {
             evaluationBarCtx.fillRect(0, 0, 30, 720);            
-
             evaluationBarCtx.fillStyle = "#ffffff";
-            evaluationBarCtx.fillText(evaluationText, 15 - evaluationTextWidth / 2, 20, 30);
+            evaluationBarCtx.fillText(evaluationText, 15 - evaluationTextWidth / 2, boardFlipped? 710 : 20, 30);
```
Comment (Levsha1337) [inline src/public/pages/report/scripts/evalbar.ts:35]: Same here: space before question mark

**Review by Levsha1337 — COMMENTED** [review]
Probably you need some codestyle fixes before that PR will be merged...

**Comment by WintrCat** [comment]
Have tested this code and it seems to be working, thank you! But as levsha pointed out, implement the codestyle fixes, and then I will merge your PR 👍

**Comment by debariev** [comment]
> Probably you need some codestyle fixes before that PR will be merged...

Thanks. Just starting with a TypeScript. Made proposed codefixes

**Pull request merged**